code42.json

```diff
@@ -75,26 +93,28 @@
                     "allow_list": true
                 },
                 "start_time": {
-                    "data_type": "numeric",
-                    "order": 1,
-                    "description": "Start of time range, in epoch time (milliseconds)",
-                    "verbose": "If not specified, the default is past 10 days"
+                    "description": "The start time to use in the initial poll in yyyy-MM-dd HH:MM:SS format (defaults to 30 days back)",
+                    "verbose": "Configure this parameter via the `start_date` configuration parameter.",
```

Comment:
Could we specify that this configuration is on the asset? `via the start_date asset configuration parameter` or something like that? 

Though I suppose this text will never be displayed on UI so perhaps it's not necessary...

Reply:
It does get displayed in the UI! And yes, I could include that it's the asset